tests/ChatTests.hs

```diff
@@ -2045,6 +2053,394 @@ testAutoReplyMessage = testChat2 aliceProfile bobProfile $
           alice <# "@bob hello!"
       ]
 
+testConnectIncognitoInvitationLink :: IO ()
+testConnectIncognitoInvitationLink = testChat3 aliceProfile bobProfile cathProfile $
+  \alice bob cath -> do
+    alice #$> ("/incognito on", id, "ok")
+    bob #$> ("/incognito on", id, "ok")
+    alice ##> "/c"
+    inv <- getInvitation alice
+    bob ##> ("/c " <> inv)
+    bob <## "confirmation sent!"
+    bobIncognito <- getTermLine bob
```

Comment:
not sure how it works - it reads the whole line, but how does it parse?

Reply:
I got it - it uses testView hack. Why not just parse the name out of string - it's everything until ':' character + extra line, just make a function getConnectedName that parses the string and you won't need the hack - it would test the actual code and not something else.